scripts/exglobal_fcst_nemsfv3gfs.sh

```diff
@@ -410,26 +411,32 @@ fi
 
 #-------------wavewave----------------------
 if [ $cplwav = ".true." ]; then
-#-------------wavewave----------------------
 
-  for file in $(ls $ROTDIR/${CDUMP}wave.${PDY}/${cyc}/rundata/rmp_src_to_dst_conserv_*) ; do
+  # Link WW3 files
+  for file in $(ls $COMINwave/rundata/rmp_src_to_dst_conserv_*) ; do
     $NLN $file $DATA/
   done
-  $NLN $ROTDIR/${CDUMP}wave.${PDY}/${cyc}/rundata/ww3_multi.${CDUMP}wave${WAV_MEMBER}.${cycle}.inp $DATA/ww3_multi.inp
+  $NLN $COMINwave/rundata/ww3_multi.${CDUMPwave}${WAV_MEMBER}.${cycle}.inp $DATA/ww3_multi.inp
 
+  # Check for expected wave grids for this run
   array=($WAVECUR_FID $WAVEICE_FID $WAVEWND_FID $waveuoutpGRD $waveGRD $waveesmfGRD $wavesbsGRD $wavepostGRD $waveinterpGRD)
   grdALL=`printf "%s\n" "${array[@]}" | sort -u | tr '\n' ' '`
+
   for wavGRD in ${grdALL}; do
-    $NLN $ROTDIR/${CDUMP}wave.${PDY}/${cyc}/rundata/${CDUMP}wave.mod_def.$wavGRD $DATA/mod_def.$wavGRD
+    # Wave IC (restart) file must exist for warm start on this cycle, if not wave model starts from flat ocean
```

Comment:
I removed these comments when I cleaned up the forecast script and added restart capability.  I am not sure why these comments need to be restored.  They are not entire accurate and necessary, and make the script very messy.  Offline documents should be developed if needed.

Reply:
Will remove.